src/integration-test/java/com/scalar/db/storage/cosmos/CosmosIntegrationTest.java

```diff
@@ -63,11 +62,6 @@ public void tearDown() {
     }
   }
 
-  // FIXME Ignore this test for now. It seems like the order of the result is not correct
-  @Ignore
-  @Override
-  public void put_MultiplePutWithDifferentConditionsGiven_ShouldStoreProperly() {}
```

Comment:
Just a question. It seems not related to the scan ordering issue, but is it OK to remove it?

Reply:
In Cosmos DB, this test failed due to the incorrect order of the result. After making the change for this test, it passes.